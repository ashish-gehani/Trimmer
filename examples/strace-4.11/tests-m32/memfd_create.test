#!/bin/sh

# Check memfd_create syscall decoding.

. "${srcdir=.}/init.sh"

run_prog
run_strace -e memfd_create $args
match_grep

exit 0
