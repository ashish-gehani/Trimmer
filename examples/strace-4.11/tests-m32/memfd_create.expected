memfd_create\("strace", MFD_CLOEXEC\|MFD_ALLOW_SEALING\|0x4\) += -1 .*
